Resources/Textures/_RMC14/Objects/Medical/first_aid_kits.rsi/meta.json

```diff
@@ -1,7 +1,7 @@
 {
   "version": 1,
   "license": "CC-BY-SA-3.0",
-  "copyright": "Taken from cmss13 at https://github.com/cmss13-devs/cmss13/blob/ca94d2e8715b73103fa9f213be53d343359b4107/icons/obj/items/storage.dmi, https://github.com/cmss13-devs/cmss13/blob/master/icons/mob/humans/onmob/inhands/equipment/medical_righthand.dmi, https://github.com/cmss13-devs/cmss13/blob/master/icons/mob/humans/onmob/inhands/equipment/medical_lefthand.dmi",
+  "copyright": "Taken from cmss13 at https://github.com/cmss13-devs/cmss13/blob/ca94d2e8715b73103fa9f213be53d343359b4107/icons/obj/items/storage.dmi, https://github.com/cmss13-devs/cmss13/blob/master/icons/mob/humans/onmob/inhands/equipment/medical_righthand.dmi, https://github.com/cmss13-devs/cmss13/blob/master/icons/mob/humans/onmob/inhands/equipment/medical_lefthand.dmi, https://github.com/cmss13-devs/cmss13-pve/blob/master/icons/obj/items/storage/medical.dmi",
```

Comment:
copyright source should include the commit ID like this picture - otherwise whenever this file is changed your source is now invalid

<img width="584" height="42" alt="image" src="https://github.com/user-attachments/assets/0deae76a-dcfb-4d0c-a5d4-d07853e780fb" />

Also not relevant to you but verm added those other links and made the same mistake